.pipelines/e2e.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       set -o pipefail
 
       . secrets/env
-      export HIVE_KUBE_CONFIG_PATH="secrets/e2e-aks-kubeconfig"
+      export HIVE_KUBE_CONFIG_PATH="secrets/aks.kubeconfig"
```

Comment:
I guess we use `aks.kubeconfig` filename everywhere, but not in the storage account (e2e-aks-kubeconfig is used in the SA).
So I suggest changing the filename in the SA. Hence we should update the e2e config.

Reply:
Please provide more context, as this isn't just a docs update if it's editing our e2e pipeline

Reply:
@cblecker thanks, I have updated the PR description

Reply:
@petrkotas Is this correct? This was introduced here from what I can tell: https://github.com/Azure/ARO-RP/pull/2362

and without updating the secrets bundle, this would break things, correct?

Reply:
yes and at the same time @yehornaumenko is right, we definitely need to chose one name. I would be in favor of having it `aks.kubeconfig` because it is visible from the context of account logged in to which I talk to.

Reply:
I have just updated the secrets bundle. So at this moment we have 2 kubeconfig files there – an old one (e2e-aks-kubeconfig) and a new one (aks.kubeconfig).

Reply:
as per https://github.com/Azure/ARO-RP/pull/2568 this variable needs to end with `_1` otherwise it will fail in e2e, because we need to select shard `1`